src/igvfd/audit/item.py

```diff
@@ -78,6 +78,8 @@ def audit_item_status(value, system):
     linked = set()
 
     for schema_path in context.type_info.schema_links:
+        if schema_path == 'supersedes':
```

Comment:
This audit is almost entirely ported from ENCODE and this is how they cased it.
https://github.com/ENCODE-DCC/encoded/blob/dev/src/encoded/audit/item.py#L184-L189